website/docs/add-secure-apps/flows-stages/stages/authenticator_email/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Email authenticator setup stage
```

Comment:
```suggestion
title: Email Authenticator Setup stage
```

Reply:
I'll add this new decision to our Docs Style Guide, but I think we should capitalize the names of exact stages.